app/assets/javascripts/base.js.erb

```diff
@@ -677,43 +677,12 @@ var instantClick
     InstantClick.init();
   }
 
+// INITIALIZE/ERROR HANDLING
 
-  // INITIALIZE/ERROR HANDLING
+  Honeybadger.configure({
```

Comment:
I was under the impression that this would be free with the deployment tracking but I am not sure. What do you think about skipping this for now and seeing what the errors look like before adding revisions?